docs/source/cpp/compute.rst

```diff
@@ -312,6 +312,40 @@ precision of `divide` is at least the sum of precisions of both operands with
 enough scale kept. Error is returned if the result precision is beyond the
 decimal value range.
 
+Trigonometric functions are also supported, and also offer ``_checked``
+variants that check for domain errors if needed.
```

Comment:
Should we add a `Trigonometric functions` subsection? Even though they are arithmetic functions, this will add a link in docs for quick access.

Reply:
I went ahead and added a subsection.